exchanges/orderbook/orderbook.go

```diff
@@ -216,7 +216,7 @@ func (b *Base) Verify() error {
 	// level books. In the event that there is a massive liquidity change where
 	// a book dries up, this will still update so we do not traverse potential
 	// incorrect old data.
-	if len(b.Asks) == 0 || len(b.Bids) == 0 {
+	if (len(b.Asks) == 0 || len(b.Bids) == 0) && !b.Asset.IsOptions() {
```

Comment:
Also getting warnings for futurecombo pairs: Potential book issue for exchange Deribit pair ETH-FS-28MAR25_26JUL24 asset future_combo length Bids 0 length Asks 0

Which matches the front-end:
![image](https://github.com/thrasher-corp/gocryptotrader/assets/4685270/10cda541-adf6-4a8f-9101-5d90bdba1002)

Reply:
I personally think that such a warning is fine for that scenario. An options pair is typically one specific type eg `BTC-USD-240607-56000-C` with only one ask and zero bids (because no one has chosen to purchase the option) and so the warning doesn't really apply. FuturesCombo while still weird, does represent a product that can have an orderbook.
I could update this warning code to not warn when both sides are zero and not warn if one side is zero and is options?
eg
```go 
if (len(b.Asks) == 0 && len(b.Bids) > 0 || len(b.Bids) == 0 && len(b.Asks) > 0) &&
		!b.Asset.IsOptions() {
```